### PR TITLE
VueUI Chemical Dispenser

### PR DESCRIPTION
**Pull request by MarinaGryphon**
* Tweaks the numeric VueUI input to look more like the old NanoUI equivalent, with one plus/minus sign per power of 10 (+, ++, +++ for 1, 10, 100) instead of just identical buttons side-by-side.
* Includes core-js for IE/ECMAscript compatibility shims.
* Converts the chemical dispenser UI to VueUI.
* Adds a numeric input to the chemical dispenser.

### CODE REVIEW

**Comment by MarinaGryphon** [comment]
!review

**Review by Geevies — APPROVED** [review]
ui code looks good, need someone skilled at vue to confirm the backend stuff

**Review by skull132 — APPROVED** [review]

File: vueui/src/main.js [hunk 1]
```diff
@@ -2,6 +2,7 @@
  * Vue.js based ui framework for SS13
  * Made for Aurora, by Karolis K.
  */
+import "core-js/stable"
```
Comment (skull132) [inline vueui/src/main.js:5]: wot is this.

Comment (MarinaGryphon) [reply vueui/src/main.js:5]: It's for `.repeat()`, it's a shim/compatibility library. Karolis suggested I include it instead of writing my own shim.

**Pull request merged**